scripts/import/import_EPMC.pl

```diff
@@ -235,12 +242,49 @@ sub import_citations{
                 -adaptor  => $pub_ad
                 );
         
-            $pub_ad->store( $publication);
-        }
-    }
+            $pub_ad->store( $publication,$source_attrib_id );
+        } 
+    } 
     close $not_found;
 }
 
+sub get_source_attrib_id{
```

Comment:
Could the look up of source_attrib_id be a single query using the query in $sql with condition t.code = 'citation_source' or did you want to report no types of citation_source separately?

Reply:
My intention was to have a separate query to get the type of citation_source. It can be a single query but if I want to use the same query in the future, it would be easier to have it separately.